src/yz_fuse.erl

```diff
@@ -26,14 +26,22 @@
 -export([create/1, check/1, check_all_fuses_ok/1, melt/1, remove/1, reset/1]).
 
 %% helpers
--export([fuse_context/0]).
+-export([fuse_context/0, encode_fuse_name/2, decode_fuse_name/1,
+         fuse_name_for_index/1]).
```

Comment:
Do we need to export encode/decode fuse name?  They only seem to be used internally.  I see that fuse_name_for_index is used in the EQC test.  Can we maybe put that in a separate export with a comment that makes it clear that the function is only used in a test?